runbot/models/build.py

```diff
@@ -1125,6 +1126,9 @@ def _cmd(self, python_params=None, py_version=None, local_only=True, sub_command
 
         faketime = []
         if faketime_params := self.params_id.config_data.get('faketime'):
+            if self.parent_id:
+                parent_time_offset = self.create_date - self.parent_id.create_date
```

Comment:
In a case where we have a lot of queue (nightly) the test could take a long time to start (bot main build and child)

This could lead to varying times for self.create_date - self.parent_id.create_date

Maybe that build_start  would be more reliable.
```suggestion
                parent_time_offset = self.create_date - self.parent_id.build_start  
```